src/EnergyPlus/UnitarySystem.cc

```diff
@@ -13972,7 +13974,9 @@ namespace UnitarySystems {
                                                                                         HVAC::CompressorOp::On);
                                     };
                                     General::SolveRoot(state, HumRatAcc, MaxIte, SolFla, SpeedRatio, f, 0.0, 1.0);
+                                    CycRatio = 0.0;
                                 }
+                                PartLoadFrac = SpeedRatio;
```

Comment:
Now I reset the PRL in one place using the `SpeedRatio` and removed the previous code changes.